Content.Server/GameTicking/Rules/NukeopsRuleSystem.cs

```diff
@@ -990,6 +994,15 @@ public void MakeLoneNukie(EntityUid mindId, MindComponent mind)
 
         //ok hardcoded value bad but so is everything else here
         _roles.MindAddRole(mindId, new NukeopsRoleComponent { PrototypeId = NukeopsId }, mind);
+        foreach (var (nukeops, gameRule) in EntityQuery<NukeopsRuleComponent, GameRuleComponent>())
+        {
+            if (mind.CurrentEntity != null)
```

Comment:
Should this if check be lifted out of the loop?

Reply:
Sure it might save a few comparisons, i don't think the foreach does a lot of iterations in that case but it can move out.
Thank you for all the review points!